crates/ilp-node/tests/redis/redis_helpers.rs

```diff
@@ -13,17 +13,23 @@ use std::time::Duration;
 
 #[allow(unused)]
 pub fn connection_info_to_string(info: ConnectionInfo) -> String {
-    match info.addr.as_ref() {
-        ConnectionAddr::Tcp(url, port) => format!("redis://{}:{}/{}", url, port, info.db),
+    match info.addr {
+        ConnectionAddr::Tcp(url, port) => format!("redis://{}:{}/{}", url, port, info.redis.db),
         ConnectionAddr::Unix(path) => {
-            format!("redis+unix:{}?db={}", path.to_str().unwrap(), info.db)
+            format!("redis+unix:{}?db={}", path.to_str().unwrap(), info.redis.db)
         }
+        // FIXME: no idea what this is used for
+        ConnectionAddr::TcpTls {
+            host,
+            port,
+            insecure,
+        } => todo!(),
```

Comment:
```suggestion
        // FIXME: no idea what this is used for
        x => unimplemented!("unsupported {:?}", x),
```

The above could work. However sadly the redis_helpers has been copypasted around, which reminds me I never remembered to PR that refactoring in which would had made these into on. Perhaps leave these as is, you probably marked all of these as FIXME and we'll create an issue on unifying these?

Reply:
I marked the two I added/touched during the upgrade as FIXME -- so if there is other similar code around then I didn't mark that.

Reply:
Created the #731 for this so resolving.